ls1intum/Artemis
# `Exam mode`: Improve action details in live statistics

### Checklist
#### General


#### Client

### Motivation and Context



Since the text doesn't look as nice as badges, I replaced the text with badges and icons.

### Description


- Replaced text with badges

### Steps for Testing

Prerequisites:
- 1 Instructor
- One/Multiple Student
- 1 Exam

1. Log in to Artemis
2. Create an Exam with Exam Live Statistics enabled
3. Participate with the student
4. Verify that the exercise section and activity section shows the correct data
5. Verify that the badges work as expected

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots



![badges](https://user-images.githubusercontent.com/60005702/179358425-45848d4d-456f-4ee6-a367-c4f72e1556a0.png)
![type](https://user-images.githubusercontent.com/60005702/179358427-e2d9aec9-62c8-4096-9ece-6029433fc0b7.png)
